src/Components/HomePage.js

```diff
@@ -8,21 +8,24 @@ import bodyweight from '../../assets/HomePage/bodyweight.jpg';
 import gymstrong from '../../assets/HomePage/gymstrong.jpg';
 import leanfit from '../../assets/HomePage/leanfit.jpg';
 import simulator from '../../assets/HomePage/simulator.jpg';
+import dictionary from '../../assets/HomePage/dictionary.jpg';
+import ourPlays from '../../assets/HomePage/ourplays.jpg';
+import essential from '../../assets/HomePage/essential.jpg';
 import huddle from '../../assets/HomePage/huddle.png';
 import { DrillTypes } from '../Fixtures/config';
 
 import { createMaterialBottomTabNavigator } from '@react-navigation/material-bottom-tabs';
-import { MaterialCommunityIcons } from '@expo/vector-icons';
+import { MaterialCommunityIcons, Ionicons } from '@expo/vector-icons';
 
 const Tab = createMaterialBottomTabNavigator();
 
 const HomeScreen = props => {
   return (
     <View style={styles.mainContainer}>
       <TouchableHighlight onPress={() => props.navigation.navigate('AnimationEditorPage')} style={styles.menuItem}>
-        <ImageBackground source={simulator} style={styles.image}>
+        <ImageBackground source={ourPlays} style={styles.image}>
```

Comment:
Maybe this should be the third item, because it is less important than drills and programs?